src/instance/read.jl

```diff
@@ -98,6 +98,10 @@ function _from_json(json; repair = true)
         json["Parameters"]["Power balance penalty (\$/MW)"],
         default = [1000.0 for t in 1:T],
     )
+    shortfall_penalty = timeseries(
+        json["Parameters"]["Reserve shortfall penalty (\$/MW)"],
+        default=[0. for t in 1:T]
+    )
```

Comment:
We don't currently describe "Reserve shortfall penalty" in the docs. Could you please add it?